src/main/java/equipments/technology/TechnologyEquipmentFactory.java

```diff
@@ -0,0 +1,20 @@
+package equipments.technology;
+
+import equipments.EquipmentType;
+
+public class TechnologyEquipmentFactory {
+
+
+    public AbstractTechnologyEquipment getTechnologyEquipment(EquipmentType technologyEquipmentType, String IP) {
+        switch (technologyEquipmentType) {
+            case VOIP:
+                return new Voip(IP);
+            case InternetComputer:
+                return new InternetComputer(IP);
+            case ArmyComputer:
+                return new ArmyComputer(IP);
+        }
+        return null;
```

Comment:
if this can take values such as clothing Equipment, don't just silently fail- throw a relevant `Exception`!